File: web/src/data/health-report-data.js
export const generateHealthReport = (userData) => ({
  userInfo: {
    name: userData?.userName || 'Test User',
    date: new Date().toLocaleDateString(),
    reportId: `LGR-${Math.random().toString(36).substr(2, 9).toUpperCase()}`
  },
  vitals: {
    temperature: {
      average: "36.5°C",
      min: "36.2°C",
      max: "36.8°C",
      status: "Normal"
    },
    bloodPressure: {
      average: "120/80 mmHg",
      min: "115/75 mmHg",
      max: "125/85 mmHg",
      status: "Normal"
    },
    heartRate: {
      average: "72 bpm",
      min: "65 bpm",
      max: "80 bpm",
      status: "Normal"
    },
    activityLevel: {
      average: "200 steps",
      min: "100 steps",
      max: "300 steps",
      status: "low"
    }
  },
  environmentalMetrics: {
    airQuality: {
      average: "75 AQI",
      status: "Moderate",
      pollutants: {
        pm25: "15.2 µg/m³",
        pm10: "45.8 µg/m³",
        no2: "25.4 ppb"
      }
    },
    humidity: {
      average: "58.8%",
      status: "Optimal"
    },
    pressure: {
      average: "1013 hPa",
      status: "Normal"
    }
  },
  activityMetrics: {
    dailySteps: {
      average: "8,500",
      goal: "10,000",
      status: "Good"
    },
    caloriesBurned: {
      average: "2,150 kcal",
      goal: "2,500 kcal",
      status: "On Track"
    },
    activeMinutes: {
      average: "45 mins",
      goal: "60 mins",
      status: "Improving"
    }
  },
  recommendations: [
    "Maintain regular physical activity levels",
    "Consider increasing daily water intake",
    "Monitor PM2.5 exposure during outdoor activities",
    "Keep up with the consistent sleep schedule"
  ]
}); 

File: mobile/lib/data/health_report_data.dart
Map<String, dynamic> generateHealthReport(Map<String, dynamic>? userData) {
  return {
    'userInfo': {
      'reportId': 'HR${DateTime.now().millisecondsSinceEpoch}',
      'date': DateTime.now().toIso8601String(),
      'name': 'Evans Acheampong',
    },
    'vitals': {
      'temperature': {
        'average': '36.5°C',
        'min': '36.2°C',
        'max': '37.1°C',
        'status': 'Normal'
      },
      'heartRate': {
        'average': '72 bpm',
        'min': '65 bpm',
        'max': '85 bpm',
        'status': 'Good'
      },
      'activityLevel': {
        'average': '6,248 steps',
        'min': '4,500 steps',
        'max': '8,000 steps',
        'status': 'Active'
      }
    },
    'environmentalMetrics': {
      'airQuality': {
        'average': '75 AQI',
        'status': 'Moderate'
      },
      'humidity': {
        'average': '65%',
        'status': 'Normal'
      },
      'pressure': {
        'average': '1013 hPa',
        'status': 'Stable'
      }
    },
    'recommendations': [
      'Consider increasing daily water intake',
      'Maintain current activity level',
      'Monitor ambient temperature during peak hours',
      'Keep up the good work with your exercise routine'
    ]
  };
}
